tests/framework/installer/ceph_manifests.go

```diff
@@ -1865,6 +1865,11 @@ func (m *CephManifestsMaster) GetRookCluster(settings *clusterSettings) string {
 		store = `storeType: "` + settings.StoreType + `"`
 	}
 
+	crushRoot := "# crushRoot not specified; Rook will use `default`"
```

Comment:
Per the previous comment, the test pools would all need to consistently use the same crush root. In this case, we likely should add a new field to the struct that can be used consistently when returning the cluster, pool, or other CRs:
```
type CephManifestsMaster struct {
	imageTag string
        crushRoot string
}
```